docs/topics/native/native-ios-integration.md

```diff
@@ -48,9 +48,14 @@ The resulting output:
 ```text
 init on <_NSMainThread: 0x600003bc0000>{number = 1, name = main}
 shared.SwiftExample
-deinit on <NSThread: 0x600003b9b900>{number = 7, name = (null)}
+deinit on <_NSMainThread: 0x600003bc0000>{number = 1, name = main}
 ```
 
+* If Swift/Objective-C objects are passed to Kotlin on a thread other than main or if the main dispatch queue isn't processed,
+  deinit on the Swift/Objective-C objects is called on a special GC thread.
```

Comment:
Also, WDYT if we use "deinitialization" word instead of "deinit"?

https://docs.swift.org/swift-book/documentation/the-swift-programming-language/deinitialization/